docs/framework/wcf/configuring-services-using-configuration-files.md

```diff
@@ -85,7 +85,7 @@ Configuring a Windows Communication Foundation (WCF) service with a configuratio
   
 -   `contract`. Specifies the interface that defines the contract. This is the interface implemented in the common language runtime (CLR) type specified by the `name` attribute of the `service` element.  
   
--   [\<endpoint> element reference](https://msdn.microsoft.com/library/13aa23b7-2f08-4add-8dbf-a99f8127c017)  
+-   [\<endpoint> element reference](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/ms731320(v=vs.100))  
```

Comment:
Maybe it would be better to link to the two endpoint elements that the topic used to link to:
https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/endpoint-element and https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/endpoint-of-client